File: Searching/Find Three Largest Number/O(n)_time_and_O(1)_space.js
function findThreeLargestNumbers(array) {
    // Write your code here.
    let maxxx = -Infinity
    let maxx = -Infinity
    let max = -Infinity
  
    for (let val of array){
      if(val>maxxx){
        max=maxx
        maxx=maxxx
        maxxx=val
      } else if(val>maxx){
        max=maxx
        maxx=val
      } else if(val>max){
        max=val
      }
    }
    return [max,maxx,maxxx]
  }

File: Searching/Find Three Largest Number/O(n)_time_and_O(1)_space.py
def findThreeLargestNumbers(array):
    # Write your code here.
    max = secMax = thirdMax = float('-inf')
    for val in array:
        if val>max:
            thirdMax=secMax
            secMax=max
            max=val
        elif val>secMax:
            thirdMax=secMax
            secMax=val
        elif val>thirdMax:
            thirdMax=val
    return[thirdMax,secMax,max]